snap/local/stage/bin/otbr-setup.sh

```diff
@@ -30,6 +30,8 @@
 
 echo "OTBR oneshot setup service"
 
+# Create directory for the OpenThread Posix Config Domain Socket
+# This set as a build flag when building OTBR
```

Comment:
`This is set...`
or rather
`This path is set with a build flag when building OTBR`